tests/FuzzFailing.elm

```diff
@@ -107,7 +107,7 @@ testOxfordify =
             , test "runs a Debug.crash on purpose" <|
                 \() ->
                     oxfordify "Everything is normal"
-                        |> Debug.crash "this test runs a Debug.crash on purpose!"
+                        |> Debug.todo "this test runs a Debug.crash on purpose!"
```

Comment:
Let's also change the name in the string literal.

Reply:
Fixed in 3553345